test/dependency/CMakeLists.txt

```diff
@@ -1,19 +1,19 @@
-cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.14...3.20)
 
 project(
   dependency
   VERSION 1.2.3
   LANGUAGES CXX
 )
 
-add_library(dependency source/dependency.cpp)
+add_library(dependency STATIC source/dependency.cpp)
```

Comment:
Why add the `STATIC` tag here?

Reply:
Because the is no export declaration used in this library.